anime-list-master.xml

```diff
@@ -2391,7 +2391,7 @@
   <anime anidbid="576" tvdbid="78923" defaulttvdbseason="0" episodeoffset="9" tmdbid="" imdbid="tt0095448">
     <name>Kimagure Orange Road: Ano Hi ni Kaeritai</name>
   </anime>
-  <anime anidbid="578" tvdbid="unknown" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="578" tvdbid="movie" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="">
```

Comment:
Since there is no ID, mark it as the AniDB type, in this case `OVA`

Reply:
Corrected with IDs for https://www.themoviedb.org/movie/66045